content/applications/inventory_and_mrp/inventory/product_management/product_tracking/serial_numbers.rst

```diff
@@ -72,222 +120,245 @@ Once the :guilabel:`Lot/Serial Number` is generated, click the blank field next
-:guilabel:`Operation Type`.
-
-For the :guilabel:`Receipts` operation type, found on the :guilabel:`Operations Types` page, the
-:guilabel:`Use Existing Lots/Serial Numbers` option can be enabled, by selecting
-:guilabel:`Receipts` from the :guilabel:`Operations Types` page, clicking :guilabel:`Edit`, and then
-clicking the checkbox beside the :guilabel:`Use Existing Lots/Serial Numbers` option (in the
-:guilabel:`Traceability` section). Lastly, click the :guilabel:`Save` button to save the changes.
-
-For the :guilabel:`Delivery Orders` operation type, located on the :guilabel:`Operations Types`
-page, the :guilabel:`Create New Lots/Serial Numbers` option can be enabled, by selecting
-:guilabel:`Delivery Orders` from the :guilabel:`Operations Types` page, clicking :guilabel:`Edit`,
-and clicking the checkbox beside the :guilabel:`Create New Lots/Serial Numbers` option (in the
-:guilabel:`Traceability` section). Be sure to click :guilabel:`Save` to save changes.
-
-.. image:: serial_numbers/serial-numbers-operations-types.png
+.. image:: serial_numbers/moves-button.png
    :align: center
-   :alt: Enabled traceability setting in operations type form.
+   :alt: Show the detailed Moves page.
 
-Serial number traceability
-==========================
+When finished, click the receipt/delivery order's breadcrumbs, and the assigned serial numbers
+will be automatically saved.
 
-Manufacturers and companies can refer to the traceability reports to see the entire lifecycle of a
-product: where it came from (and when), where it was stored, and who it went to.
+Traceability & Reporting
+========================
+
+Manufacturers and companies can refer to the :guilabel:`Lots/Serial Numbers` dashboard and
+traceability reports to see the entire lifecycle of a product: when and where it originated,
+where it was stored, and who it was shipped to.
+
+Lots/Serial Numbers dashboard
+-----------------------------
 
 To see the full traceability of a product, or group by serial numbers, go to
 :menuselection:`Inventory app --> Products --> Lots/Serial Numbers`. Doing so reveals the
 :guilabel:`Lots/Serial Numbers` dashboard.
 
-From here, products with serial numbers assigned to them will be listed by default, and can be
-expanded to show what serial numbers have been specifically assigned to them.
+Reporting
+---------
+
+On the :guilabel:`Lots/Serial Numbers` dashboard, products with serial numbers assigned to them are
+listed by default. Click the :icon:`fa-caret-right` :guilabel:`(expand)` icon to show which serial
+numbers are assigned to the chosen product.
 
 To group by serial numbers (or lots), first remove any default filters from the search bar in the
 upper-right corner. Then, click :guilabel:`Group By`, and select :guilabel:`Add Custom Group`, which
 reveals a mini drop-down menu. From this mini drop-down menu, select :guilabel:`Lot/Serial Number`,
 and click :guilabel:`Apply`.
```

Comment:
Slight update to this, since there is nowhere to click "Group By"
```suggestion
upper-right corner. Then, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` icon and select
:guilabel:`Add Custom Group`, which reveals a mini drop-down menu. From this mini drop-down menu,
select :guilabel:`Lot/Serial Number`, and click :guilabel:`Apply`.
```